src/zenml/post_execution/pipeline_run.py

```diff
@@ -160,6 +160,23 @@ def enable_artifact_metadata(self) -> Optional[bool]:
             PARAM_ENABLE_ARTIFACT_METADATA
         )
 
+    @property
+    def commit(self) -> Optional[str]:
```

Comment:
Do we want this part of `StepContext` as well?

Reply:
Mhm is this useful enough to access this while a step is being executed to warrant a separate property? I guess I would rather expose the entire `PipelineRunView` instead of duplicating all the potentially useful stuff we expose there?

Reply:
Yeah youre right, we already have that sort of ticket in FRW: https://zenml.atlassian.net/browse/FRW-1053

Reply:
Do you think it's okay if we leave out the `commit` property from the step context then until we tackle that ticket?

Reply:
Yes